test/fuzz/lua/test_engine.lua

```diff
@@ -1406,6 +1406,7 @@ end
 local function run_test(num_workers, test_duration, test_dir,
                         engine_name, verbose_mode)
 
+    fio.mkdir(fio.abspath(test_dir))
```

Comment:
Should it be a part of this patch?
Should this dir be removed later if the test finishes successfully?

Reply:
It is already fixed by Volodya in master branch. I'll revert it.

Reply:
```diff
--- a/test/fuzz/lua/test_engine.lua
+++ b/test/fuzz/lua/test_engine.lua
@@ -1406,7 +1406,6 @@ end
 local function run_test(num_workers, test_duration, test_dir,
                         engine_name, verbose_mode)
 
-    fio.mkdir(fio.abspath(test_dir))
     local socket_path = fio.pathjoin(fio.abspath(test_dir), 'console.sock')
     console.listen(socket_path)
     log.info(('console listen on %s'):format(socket_path))
```

reverted